test/Makefile

```diff
@@ -0,0 +1,85 @@
+#
+# SYNOPSIS:
+#
+#   make [all]  - makes everything.
+#   make TARGET - makes the given target.
+#   make clean  - removes all files generated by make.
+
+# Points to the root of Google Test source
+GTEST_DIR = /usr/src/googletest/googletest
```

Comment:
This would need to be detected by `configure`, can't have it hardcoded like that.

Reply:
Absolutely... do you think there should be a separate `configure` here in the `test` directory, or that this Makefile should be generated by the main `configure`, behind some flag that isn't enabled by default?

Reply:
Probably the usual with-XXX autodetection yes